wesnoth-units.po

```diff
@@ -3061,19 +3045,19 @@ msgid ""
 "Elvenkind."
 msgstr ""
 "Vynikající elfí válečníci jsou někdy nazýváni „Sárcan“, což přibližně "
-"znamená „ten, kdo se obětoval na obranu“ nebo „proměněný zkušeností“. Toto "
-"jméno, do lidské řeči často překládáno jako „elfí hrdinové“, naznačuje "
-"určitou úroveň vážnosti a významnosti, ale nepoužívá se k oslavě postavení, "
-"které tito elfové zastávají. Elfí hrdinové jsou ti, kteří přestáli mnohé "
-"války a přežili mnoho svých přátel, již padli v předchozích bitvách. "
-"Spatřili všechny způsoby krutosti páchané každou rasou, ať to byli skřeti "
-"a lidé pustošící přirozenou krásu lesů nebo dokonce jejich vlastní druh, "
-"jiní elfové, kteří se nepovznesli nad vraždění nevinných lidí z pýchy a pro "
-"pomstu. Mnoho let bojů z těchto šermířů přirozeně učinilo velké odborníky na "
-"boj zblízka, ale útrapy a ničení působené válkou si vybírají velkou daň na "
-"duši i na těle. Přestože si elfové uvědomují nezbytnost vojenských sil a ctí "
-"„Sárcanet“ za jejich službu jako hrdiny, není toto postavení většinou elfího "
-"národa často vyhledáváno."
+"znamená „ten, kdo se obětoval na obranu“ nebo „ten, kdo byl proměněn "
+"zkušeností“. Toto jméno, které je v jazyku lidí často překládáno jako „elfí "
+"hrdinové“, naznačuje určitou úroveň proslulosti, ale není určeno k oslavě "
+"této pozice. Elfí hrdinové jsou ti, kteří přestáli mnohé války a přežili "
+"mnoho svých přátel, co padli v předchozích bitvách. Tito bojem zakalení "
+"vojáci, kteří často patří mezi několik osamělých přeživších, byli zasvěceni "
+"do četných zvěrstev války, ať to byli skřeti a lidé, pustošící přirozenou "
+"krásu jejich lesních domovů, trpaslíci, drancující vzácné artefakty od "
+"padlých, nekromanti, pozvedávající jejich vlastní druhy proti nim nebo "
+"dokonce jejich vlastní druh, zabíjející bez výčitek a milosrdenství nevinné. "
+"Přestože si elfové uvědomují nezbytnost vojenských sil a ctí „Sárcanty“ za "
```

Comment:
Řekl bych, že koncovka -et znamená v elfštině množné číslo. Pokud to chcete počeštit, skloňoval bych Sárcana podle vzoru pán.
```suggestion
"Přestože si elfové uvědomují nezbytnost vojenských sil a ctí „Sárcany“ za "
```